docs/Usage-Guide.md

```diff
@@ -1,3 +1,172 @@
+
+### (Optional) Sitefinity/WebServiceApiKey
+
+(optional) if the web service endpoint is configured to have a key, enter it in the here.
+
+### Sitefinity/ModuleDeploymentFolderPath
+
+**Absolute path** to the deployment folder created by the 'Export for Deployment' process in Sitefinity. This tool requires the deployment folder and type definitions in order to import the content types required for the content items and pages.
+
+### Sitefinity/CodeNamePrefix
+
+Codename prefix for all content types. Typically the name of your site. Do not use CMS or OM.
+
+### Sitefinity/PageContentTypes
+
+A list of types that are considered to be pages such as, news, blogs, etc. Sitefinity allows for content 'detail' pages to be displayed from a single 'listing' page. e.g. A news 'listing' page will link to a large number of news 'detail' pages. This pattern is different in XbyK. To support the strong page types in XbyK you must configure the PageContentTypes. If they are not configured, all content items will be placed in the Content Hub and will no longer have their hierarchical structure or url structure. See below for which PageTemplateType should be used.
+
+Sitefinity allows users to add a listing widget for any particular static or dynamic type. When a listing widget is added to the page, Sitefinity will consider that page url to be the root of the items that are specified for that listing. For example, a news listing widget may be on '/news'. If a user opened a news item with the urlname of '/example-1', the url would be '/news/example-1' and Sitefinity would dynamically render the page for that url. In Sitefinity, the item for '/example-1' would still live in the news content items but in XbyK, we need that content item to live under '/news'. To mimic this behavior in XbyK, you would add the following record.
+
+```
+{
+  "TypeName": "NewsItem",
+  "PageRootPath": "/news",
+  "PageTemplateType": "Listing"
+}
+```
+
+Sitefinity also allows the user to use the same widget as a detail view. This allows the content editor to select a particular item and render the content as if the structured content was part of the page itself. If the item being shown also had child items, they would typically render in a listing as part of the content. To import a content item as a detail page and the structured content to be associated with that page, the detail template type will need to be added to the configuration. For example, a user may have a page named "/blog-one" that contains the blog detail widget with the "/blog-1" content item selected. The page would render the content for "/blog-1" and show any child blog items as a listing. When navigating to those blog items, the "/blog-one" url will be prepended. For example, "/blog-one/blog-item-1". To mimic this behavior in XbyK, you would add the following record.

```

Comment:
"Prepended" is a valid word, but I'd replace it with something more understandable or reword the sentence.

Reply:
I have updated the sentence. Please let me know if the rewording resolves the issue. Thank you!

Reply:
Better now :)